martin/src/config/file/tiles/postgres/resolver/query_tables.rs

```diff
@@ -22,13 +22,23 @@ pub type SqlTableInfoMapMapMap = BTreeMap<String, BTreeMap<String, BTreeMap<Stri
 const DEFAULT_EXTENT: u32 = 4096;
 const DEFAULT_BUFFER: u32 = 64;
 const DEFAULT_CLIP_GEOM: bool = true;
+const QUERY_AVAILABLE: &str = include_str!("scripts/query_available_tables.sql");
 
 /// Queries the database for available tables with geometry columns.
-pub async fn query_available_tables(pool: &PostgresPool) -> PostgresResult<SqlTableInfoMapMapMap> {
+pub async fn query_available_tables(
+    pool: &PostgresPool,
+    filtered_tables: Option<Vec<&str>>,
+) -> PostgresResult<SqlTableInfoMapMapMap> {
+    let mut query = QUERY_AVAILABLE.to_string();
+
+    if let Some(table_names) = filtered_tables {
+        query.push_str(&format!("WHERE name in ('{}')\n", table_names.join("','")));
```

Comment:
to fix the clippy lint

```suggestion
        query.push_str("WHERE name in ('");
        query.push_str(&table_names.join("','"));
        query.push_str("')");
```